assets/pages/bundles-landing/components/Bundles.jsx

```diff
@@ -178,6 +181,7 @@ const getContribAttrs = (
 
   params.append('contributionValue', contribAmount[contType].value);
   params.append('contribType', contribType);
+  params.append('currency', currency);
```

Comment:
So, by default, we set currency on the checkouts using a query param?

Reply:
Yes, since by default we set the amount of the checkouts using a query parameter.